labs/in-cluster/deploy/k6.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: k6
+  namespace: k6
+  labels:
+    app.kubernetes.io/name: k6
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "9090"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: k6
+  template:
+    metadata:
+      labels:
+        app: k6
+    spec:
+      containers:
+        - name: app
+          env:
+          - name: "K6_PROMETHEUS_RW_SERVER_URL"
+            value: "http://prometheus-service.monitoring.svc.cluster.local:8080/api/v1/write"
+          - name: "MY_HOSTNAME"
```

Comment:
If possible, I recommend changing the name `MY_HOSTNAME` to something more descriptive. It seems like it is the target host that k6 will be testing. But in some contexts, it can read like it is the host of k6 itself.

Reply:
good point, I renamed the variable to TARGET_HOSTNAME instead.